packages/makeswift-next-plugin/index.js

```diff
@@ -58,6 +58,31 @@ module.exports =
           fallback: Array.isArray(rewrites) ? [] : rewrites?.fallback ?? [],
         }
       },
+      async headers() {
+        const headers = await nextConfig.headers?.()
+
+        return [
+          ...headers,
+          {
+            source: '/:path*',
+            has: [
+              {
+                type: 'query',
+                key: 'x-makeswift-load-fonts',
+                value: 'true',
+              },
+            ],
+            headers: [
+              {
+                key: 'Access-Control-Allow-Origin',
+                value:
+                  process.env.MAKESWIFT_APP_ORIGIN ||
```

Comment:
We probably want to make the app origin an option for the plugin, not an environment variable. I think environment variables should be handled at an application level, but this package is a library

Reply:
Okay just updated it to be an option instead of an env.